lib/api/multiObjectDelete.js

```diff
@@ -269,6 +269,8 @@ export function getObjMetadataAndDelete(authInfo, canonicalID, request,
             // return DeleteMarkerVersionId equals the versionID of the marker
             // you just generated and DeleteMarker tag equals true
             if (deleteInfo.newDeleteMarker) {
+                // Adding a delete marker should add one to numberOfObjects
+                numOfObjects--;
```

Comment:
Comment 1: Please see my response, just above, to Jonathan.
Comment 2: In this case, we would need to decrement storage utilized by the null version, if any. And it sounds like number of objects should remain unchanged (since we both delete a version, and put a delete marker).